### PR TITLE
Add Open Voice Challenge Terms & Conditions

### PR DESCRIPTION
**Pull request by rileyjshaw**

### CODE REVIEW

File: Common_Voice_Challenge_Terms/en.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+Each week during the Pilot Challenge, Mozilla will send you an email announcing the week’s task and how to win. Mozilla will donate $100 to an organization of each individual winner’s choosing (from a list of Mozilla-approved organizations). Mozilla will donate $500 to a charitable organization for each team winner’s choosing (from the same list of Mozilla-approved organizations).
+
+## 3. Communications
+
+When you sign up for the Common Voice Pilot Challenge, you may opt-in to receive emails each week about the Pilot Challenge. If opted-in, you may also receive follow up emails regarding the Pilot Challenge.
+
+## 4. Disclaimers, Governing Law, and Disputes
+
+### Disclaimer, Release and Limit of Liability.
```
Comment (peiying2) [inline Common_Voice_Challenge_Terms/en.md]: Nit pick: an extra comma after `Release` to be consistent with the style.

**Comment by peiying2** [comment]
@mafeld could you review and sign off?

**Comment by mafeld** [comment]
Approved. Thank you.

**Comment by rileyjshaw** [comment]
@peiying2 @mafeld added the comma, thanks for the review!

File: Common_Voice_Challenge_Terms/en.md (hunk 1 above)
Comment (rileyjshaw) [reply Common_Voice_Challenge_Terms/en.md]: Done 👍

**Pull request merged**